File: snippets/csharp/arrays/replace/Replace.cs

using System;

public class Replace {
	static void Main(string[] args)
	{
		int[] array = {3, 2, 1};
		int index = 1;
		int item = 100;

		Console.WriteLine("before replacing");
		foreach (int el in array)
        {
            Console.WriteLine(el);
        }
        array[index] = item;
		Console.WriteLine("after replacing");
		foreach (int el in array)
        {
            Console.WriteLine(el);
        }
	}

}

File: snippets/c++/array/replace/replace.cpp
#include<iostream>
#include<vector>

using namespace std;

int main() {
   int array[] = {3, 2, 1};

    cout<<"before replacing"<<endl;
    for (int i = sizeof(array) / sizeof(array[0])-1; i >= 0; i--) {
        cout << array[i];
    }
    cout << endl;
    int item = 100;
    int index = 1;
    array[index] = item;

    cout<<"after replacing"<<endl;
    for (int i = sizeof(array) / sizeof(array[0])-1; i >= 0; i--) {
        cout << array[i];
    }
    cout << endl;
   return 0;
}